tests/contrib/flask/test_flask_appsec.py

```diff
@@ -16,3 +16,20 @@ def test_flask_simple_attack(self):
 
         assert "triggers" in json.loads(root_span.get_tag("_dd.appsec.json"))
         assert _context.get_item("http.request.uri", span=root_span) == "http://localhost/.git?q=1"
+        assert _context.get_item("http.request.query", span=root_span)["q"] == "1"
```

Comment:
same here, any reason not to assert that it is equal to `{"q": "1"}` ?

Reply:
```suggestion
        assert _context.get_item("http.request.query", span=root_span)["q"] == {"q": "1"}
```